pkgs/applications/misc/input-leap/default.nix

```diff
@@ -1,46 +1,57 @@
-{ lib, fetchFromGitHub, cmake, curl, xorg, avahi, qt5, openssl, stdenv
+{ lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, curl
+, xorg
+, avahi
+, qtbase
+, qttools
+, wrapQtAppsHook
+, openssl
+, stdenv
 , wrapGAppsHook
 , avahiWithLibdnssdCompat ? avahi.override { withLibdnssdCompat = true; }
-, fetchpatch }:
+, fetchpatch
+}:
 
 stdenv.mkDerivation rec {
   pname = "input-leap";
-  version = "2.4.0";
+  version = "unstable-2023-02-25";
 
   src = fetchFromGitHub {
-    owner = "input-leap";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-2tHqLF3zS3C4UnOVIZfpcuzaemC9++nC7lXgFnFSfKU=";
+    owner = "KoviRobi";
+    repo = "input-leap";
+    rev = "62ff806e08bdd2a06973175cbd36f4b08b9803ef";
+    sha256 = "lrNnvIbupK19ECy47Sffjb8zR4E35GxcQTdoae1Qnd0=";
     fetchSubmodules = true;
   };
 
-  patches = [
-    # This patch can be removed when a new version of inputleap (greater than 2.4.0)
-    # is released, which will contain this commit.
-    (fetchpatch {
-      name = "add-missing-cstddef-header.patch";
-      url =
-        "https://github.com/input-leap/inputleap/commit/4b12265ae5d324b942698a3177e1d8b1749414d7.patch";
```

Comment:
This isn't required because we are using the latest. **Note however** that you have a typo, the repo name is `input-leap` not `inputleap`. So how come it worked for you eh? Well if the output with the given `sha256` exists then nix won't download it again.

This is a common gotcha for nix, and important to take on-board. It often pops up as supposing you have
```
  src = fetchFromGitHub {
    owner = "input-leap";
    repo = pname;
    rev = "v${version}";
    sha256 = "sha256-2tHqLF3zS3C4UnOVIZfpcuzaemC9++nC7lXgFnFSfKU=";
  };
```
and you realise you need to fetch submodules, so you change it to
```
  src = fetchFromGitHub {
    owner = "input-leap";
    repo = pname;
    rev = "v${version}";
    sha256 = "sha256-2tHqLF3zS3C4UnOVIZfpcuzaemC9++nC7lXgFnFSfKU=";
    fetchSubmodules = true;
  };
```
but it doesn't actually download the submodules for you, because you've previously downloaded it without the submodules and that specific `sha256` points to the one without submodules. You can use `lib.fakeSha256`, or `nix-prefetch-github foo bar --nix --fetch-submodules`